OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -259,6 +259,13 @@ public override bool Tick(Actor self)
 			return Pair.New(nextCell, subCell);
 		}
 
+		public override void Cancel(Actor self, bool keepQueue = false)
+		{
+			path.Clear();
```

Comment:
`path` can be null, according to the NRE exception that I just found on the shellmap.

Reply:
Fixed